src/worker/storage.ts

```diff
@@ -71,4 +92,27 @@ export default class Storage {
   fetchAllFiles(): Promise<StoredWordList[]> {
     return this.db.files.toArray();
   }
+  /**
+   * Update bcp47 tag to database
+   * @param bcp47Tag
+   */
+  updateBCP47Tag(bcp47Tag: string): Promise<void> {
+    /**
+     * The key of the ONLY StoredPackageInfo record.
+     */
+    const PACKAGE_ID = 0;
+    return this.db.transaction("readwrite", this.db.packageInfo, async () => {
+      await this.db.packageInfo.put({ bcp47Tag, id: PACKAGE_ID });
+    });
+  }
+  /**
+   * Retrieves packageInfo in the database
+   */
+  fetchPackageInfo(): Promise<StoredPackageInfo | undefined> {
+    /**
+     * The key of the ONLY StoredPackageInfo record.
+     */
+    const PACKAGE_ID = 0;
```

Comment:
Since the `PACKAGE_ID` constant represents the exact same value across two different methods, it should be extracted to the module scope to DRY things up:

```suggestion
  /**
   * Update bcp47 tag to database
   * @param bcp47Tag
   */
  updateBCP47Tag(bcp47Tag: string): Promise<void> {
    return this.db.transaction("readwrite", this.db.packageInfo, async () => {
      await this.db.packageInfo.put({ bcp47Tag, id: PACKAGE_ID });
    });
  }
  /**
   * Retrieves packageInfo in the database
   */
  fetchPackageInfo(): Promise<StoredPackageInfo | undefined> {
```

And then place this _above_ the `class` definition:

```typescript
/**
 * The key of the ONLY StoredPackageInfo record.
 */
const PACKAGE_ID = 0;
```